build.jam

```diff
@@ -0,0 +1,66 @@
+# Copyright René Ferdinand Rivera Morell 2023
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+project /boost/log
+    : common-requirements
+        <library>/boost/align//boost_align
```

Comment:
Same as for other built libraries, those should probably be attached to the respective libraries (which there are two for Boost.Log) and public and private dependencies separate.